utils/wrappers/aws/api_gateway.py

```diff
@@ -38,6 +38,10 @@ def wrapper(event: dict, context: LambdaContext, *args: Any, **kwargs: Any) -> d
             try:
                 api_gateway_event = ApiGatewayEvent(event, context)
                 result = function_to_call(api_gateway_event, context, *args, **kwargs)  # type: ignore
+
+                result['headers']['Access-Control-Allow-Origin'] = '*'
```

Comment:
Why are we using `*` here?

Woudn't it be better to specify the website domain? It could be configured as an EnvVar

Reply:
I'm not sure if we want to restrict it. I think we can keep it for now and discuss about it soon.